website/identifiers/tasks.py

```diff
@@ -17,3 +17,29 @@ def task__update_doi_metadata_on_change(self, target_guid):
 @celery_app.task(ignore_results=True)
 def update_doi_metadata_on_change(target_guid):
     task__update_doi_metadata_on_change(target_guid)
+
+@celery_app.task(bind=True, max_retries=5, acks_late=True)
+def task__update_doi_metadata_with_verified_links(self, target_guid):
+    sentry.log_message('Updating DOI with verified links for guid',
+                      extra_data={'guid': target_guid},
+                      level=logging.INFO)
+
+    Guid = apps.get_model('osf.Guid')
+    target_object = Guid.load(target_guid).referent
+    try:
+
+        target_object.request_identifier_update(category='doi')
+
+        sentry.log_message('DOI metadata with verified links updated for guid',
+                         extra_data={'guid': target_guid},
+                         level=logging.INFO)
+    except Exception as exc:
+        sentry.log_message('Failed to update DOI metadata with verified links',
+                         extra_data={'guid': target_guid, 'error': str(exc)},
+                         level=logging.ERROR)
+        raise self.retry(exc=exc)
```

Comment:
We now have `GVException` with we can raise and except, for example, see: https://github.com/CenterForOpenScience/osf.io/pull/11113/files#diff-d288f5a6729b0d85cc2d5c82086b82324edbb94b369fac5f719e9fb01dbfc726R103-R105